.swiftlint.yml

```diff
@@ -5,6 +5,9 @@ excluded:
 opt_in_rules:
   - empty_count
 
+type_body_lenght:
```

Comment:
This also appears to be misspelled.

Reply:
Fixed by 7338fe00e356705a3c96de51af5dce8af181fdc5